app/controllers/users_controller.rb

```diff
@@ -597,6 +597,7 @@ def edit
       format.html do
         @monthly_supporter = @user.donorbox_plan_status == "active" &&
           @user.donorbox_plan_type == "monthly"
+        render :edit2, layout:"bootstrap"
```

Comment:
Let's change this to support testing without changing the current behavior just yet, e.g.

```ruby
if is_admin? && params[:test] == "profile"
  render :edit2, layout: "bootstrap"
else
  render :edit
end
```

Reply:
Done, and unless I'm missing something else in the comments, I believe this should be the last fix.